packages/react-tinacms-editor/src/state/plugins/Menu/Menu.tsx

```diff
@@ -184,7 +176,7 @@ export const Menu = (props: Props) => {
               <ItalicControl view={view} />
               <UnderlineControl view={view} />
               <LinkControl view={view} />
-              <ImageMenu editorView={editorView} imageUpload={imageUpload} />
+              {/* <ImageMenu editorView={editorView} imageUpload={imageUpload} /> */}
```

Comment:
As discussed over slack we can create a modal in editor for image upload and then enable this menu option.